Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsFragmentPresenter.java

```diff
@@ -689,6 +689,14 @@ private void addGraphicsSettings(ArrayList<SettingsItem> sl)
             R.string.video_backend, 0, R.array.videoBackendEntries, R.array.videoBackendValues));
     sl.add(new CheckBoxSetting(mContext, BooleanSetting.GFX_SHOW_FPS, R.string.show_fps,
             R.string.show_fps_description));
+    sl.add(new CheckBoxSetting(mContext, BooleanSetting.GFX_SHOW_VPS, R.string.show_vps,
+            R.string.show_vps_description));
+    sl.add(new CheckBoxSetting(mContext, BooleanSetting.GFX_SHOW_SPEED, R.string.show_speed,
+            R.string.show_speed_description));
+    sl.add(new CheckBoxSetting(mContext, BooleanSetting.GFX_SHOW_FRAME_TIMES, R.string.show_frame_times,
+            R.string.show_frame_times_description));
```

Comment:
You forgot to change the string `R.string.show_frame_times` to `R.string.show_speed_frame_times`
`R.string.show_frame_times_description` also needs to be `R.string.show_speed_frame_times_description`
Maybe remove "speed_" in the string name?